code/processes/wdb.q

```diff
@@ -205,11 +205,17 @@ endofdaysave:{[dir;pt]
 
 /- add entries to dictionary of callbacks. if timeout has expired or d now contains all expected rows then it releases each waiting process
 handler:{
-	.wdb.d[.z.w]:x;
+	/-insert process reload outcome into .wdb.d
+	if[not .z.w in  key .wdb.d;
+        	.wdb.d[.z.w]:x];
+	if[(not .wdb.d[.z.w]`status);
+		.lg.o[`reload;string[.wdb.d[.z.w]`process]," reload ", string[.wdb.d[.z.w]`result]]];
```

Comment:
I think it's correct to log this as .lg.o rather than .lg.e
But it would be better to make the error more explicit i.e. 

"process equityrdb failed to reload with error ",error

Reply:
Have edited the structure of this so the log in .lg.o looks like
```
2022.02.23D09:28:33.441218000|satoshi|wdb|wdb1|INF|reload|the rdb process reload has failed with error: eodtabcount
```
However my error message in the result column of my .wdb.reloadstatus is quite lengthy - looks something like this:
```
handle| process status result
------| ---------------------------------------------
8     | rdb     0      failed with error: eodtabcount
```
Would it be better to shorten that message that's added to .wdb.reloadstatus result column and then edit the .lg.o statement?

Reply:
looks fine